README.md

```diff
@@ -125,7 +133,15 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+genie = Genie.create(name: "Genie")
+lamp = Lamp.create(wishes_remaining: 3, genie: genie)
+
+Lamp.update(wishes_remaining: 1)
+jafar = Genie.create(name: "Jafar")
+Lamp.create(wishes_remaining: 3, genie: jafar)
+lamp.update(wishes_remaining: 1)
+
+# This question is a little confusing.
 ```
```

Comment:
:+1:

You got it! Although for that last one you want to set the lamp's wishes to `nil`, not 1.